### PR TITLE
Code Quality: Switch back to use WinUIEx (partial revert of #15834)

### PR DESCRIPTION
**Pull request by 0x5bfa**

### CODE REVIEW

**Comment by 0x5bfa** [comment]
wait a min, im testing

File: src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs [hunk 1]
```diff
@@ -103,13 +103,15 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
...
+				propertiesWindow = new();
 				propertiesWindow.Closed += PropertiesWindow_Closed;
 			}
 
 			var width = Convert.ToInt32(800 * App.AppModel.AppWindowDPI);
 			var height = Convert.ToInt32(500 * App.AppModel.AppWindowDPI);
 
+			propertiesWindow.MinHeight = 550;
+			propertiesWindow.MinWidth = 460;
 			propertiesWindow.AppWindow.Resize(new (width, height));
```
Comment (yaira2) [inline src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:115]:
```suggestion
			propertiesWindow.AppWindow.Resize(new (800, 500));
```

File: src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs [hunk 2]
```diff
@@ -103,13 +103,15 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
 
 			if (!WindowCache.TryTake(out var propertiesWindow))
 			{
-				propertiesWindow = new(460, 550);
+				propertiesWindow = new();
 				propertiesWindow.Closed += PropertiesWindow_Closed;
 			}
 
 			var width = Convert.ToInt32(800 * App.AppModel.AppWindowDPI);
 			var height = Convert.ToInt32(500 * App.AppModel.AppWindowDPI);
```
Comment (yaira2) [inline src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:111]:
```suggestion
```

Comment (0x5bfa) [reply src/Files.App/Utils/Storage/Helpers/FilePropertiesHelpers.cs:111]: As above.

**Pull request closed without merging**